packages/core/src/textures/Texture.js

```diff
@@ -346,6 +347,12 @@ export default class Texture extends EventEmitter
 
         let texture = TextureCache[cacheId];
 
+        // Strict-mode rejects invalid cacheIds
+        if (cacheId && strict && !texture)
```

Comment:
`cacheId` getted from `source.__pixiId` but on first usage is will be undefined and assigned from `uid()` and was't exist in TextureCache

Reply:
Oops, you're right. Will fix.